pkg/cstor/pv_operation.go

```diff
@@ -158,3 +198,35 @@ func isCSIPv(pv v1.PersistentVolume) bool {
 	}
 	return false
 }
+
+// getSnapListFromCloud returns list of all the snapshots(base and incremental) associated with given volume from cloud
+func (p *Plugin) getSnapListFromCloud(vol *Volume) ([]string, error) {
```

Comment:
let us move this function to  pkg/clouduploader and pass bkpname and tag (snapshottag), which will return the snapshot list.